mne/channels/interpolation.py

```diff
@@ -104,8 +104,33 @@ def _make_interpolation_matrix(pos_from, pos_to, alpha=1e-5):
     return interpolation
 
 
+def _do_interp_dots(inst, interpolation, goods_idx, bads_idx):
+    """Dot product of channel mapping matrix to channel data
+    """
+    from mne.io.base import _BaseRaw
+    from mne.epochs import _BaseEpochs
+    from mne.evoked import Evoked
```

Comment:
I don't know. I just kept what @dengemann had originally. I suppose to avoid circular imports?